src/NuGet.Clients/PackageManagement.UI/PackageLoader.cs

```diff
@@ -485,7 +487,7 @@ private IEnumerable<string> GetSupportedFrameworks()
 
                 // if the package does not exist in the remote source, NuGet should
                 // try getting metadata from the local resource.
-                if (String.IsNullOrEmpty(metadata.Summary) && localResource != null)
+                if (String.IsNullOrEmpty(metadata?.Summary) && localResource != null)
```

Comment:
I don't know what original intention was behind this particular algorithm.
I guess by querying remote first one could get the most up-to-date metadata.
Anyway it's gonna change when all-sources feature is merged.